docs/api.rst

```diff
@@ -289,6 +289,9 @@ others fail.
 The closest to regular Python behavior is the :class:`StrictUndefined` which
 disallows all operations beside testing if it's an undefined object.
 
+When :class:`Undefined` is created in an ``except`` or ``finally`` clause,
```

Comment:
Did you mean to finish this line, or did you move it further down and forget to remove it here?